### ISSUE TITLE
reproduce Electricity data set results

### ISSUE THREAD

**Issue by MorShahrezaye** [vincent-leguen/STRIPE#1]
I tried to reproduce the reported MSE on Electricity data set. I tried various parameters but I could not get close to the MSE reported in the original paper. Can you please let me know which set of parameters you used in the paper? Or can you please upload the corresponding python file?

**Comment by Lottie215** [comment]
I have a question about how to change codes to reproduce results on the Electricity data set. Should I construct time windows? How to divide the training set and the test set? Is there a proportional relationship between the two? And is the synthetic_dataset  univariate time series? What should I change if I want to forecast a variate time series? I would be very grateful if you could resolve my doubts？Thanks.

**Comment by hamgcho** [comment]
Hello, I am in a similar situation as you've been.
I am trying to adapt this model to the custom dataset other than the synthetic dataset.

And as far as I understand, I cannot find any specific description about what kind of data preprocessing was used on the experiment with Electricity/Traffic dataset.
